### ISSUE TITLE
fail to use hsdev-0.3.4.*

### ISSUE THREAD

**Issue by LC0117** [SublimeHaskell/SublimeHaskell#454]
This plugin fails when I have hsdev's version higher than 0.3.3, is this a bug?

**Comment by st8tikratio** [comment]
Same issue here.

**Comment by jaigarV** [comment]
I am also using Sublime 4 and I got the same errors as @st8tikratio 

I went in my local system to where sublime stores downloaded packages in Linux ~/.config/sublime-text-3/Packages/SublimeHaskell and inside to the folder hsdev. I changed the constant HSDEV_MAX_VER to [0, 3, 9, 0] of the file [backend.py](https://github.com/SublimeHaskell/SublimeHaskell/blob/master/hsdev/backend.py) and it kind of works, although not the special theme.

Let's see is some of the maintainers of the repo (@holmak @nh2 @mvoidex) can check if this version is still compatible and update the code, so that people can use it directly from sublime package control because it looks like a great package.

**Comment by mvoidex** [comment]
Yes, there's no more active support. You may take a look at https://haskell-language-server.readthedocs.io/en/latest/

**Comment by LC0117** [comment]
I would recommend using neovim+LSP or vscode+haskellplugin for better experience for the situation.
